samples/project-anfield/export_cluster_config.py

```diff
@@ -84,7 +85,10 @@
     "sources": library.get_protection_sources(cohesity_client),
     "remote_clusters": library.get_remote_clusters(cohesity_client),
     "sql_entity_mapping": library.get_sql_entity_mapping(cohesity_client,
-                                                         env_enum.KSQL)
+                                                         env_enum.KSQL),
+    "ad": library.get_ad_entries(cohesity_client),
```

Comment:
Can this be config driven ? Selectively import AD only if specified in the config ?

Reply:
Updated to fetch from config file